src/pages/documentation/adobe-target/api-reference.md

```diff
@@ -229,22 +229,22 @@ React Native
 
 ## setSessionId
 
+<InlineAlert variant="info" slots="text"/>
+
+This API is only available in Android and latest iOS Target Mobile SDKs.
```

Comment:
If we're mentioning latest iOS Target Mobile SDKs, maybe it'd be good to link to it too?